lib/internal/Magento/Framework/Code/Test/Unit/Generator/EntityAbstractTest.php

```diff
@@ -57,7 +57,7 @@ class EntityAbstractTest extends TestCase
     protected function setUp(): void
     {
         $this->sourceClass = '\\' . DataObject::class;
-        $this->resultClass = '\\' . \Magento\Framework\DataObject_MyResult::class;
+        $this->resultClass = '\\Magento\\Framework\\DataObject_MyResult';
```

Comment:
why was it changed?

Reply:
Hello @ihor-sviziev 
Class `\Magento\Framework\DataObject_MyResult` doesn't exist in the system. The static tests fail because of it.
Static tests check files with changes, but not only changes.